commit 24f24e03793c8214a5d1f3414a5aeb48eccef4f4
Author: Bhushan Shah <bhush94@gmail.com>
Date:   Wed Mar 25 21:25:22 2015 +0530

    Workaround the lockscreen password field focus issue
    
    Fixes bug 344823
    Backport of a476e1b6bf6f683bd74000bb30076868c9f92371 in 5.2
    
    CCMAIL: release-team@kde.org

diff --git a/lookandfeel/contents/lockscreen/LockScreen.qml b/lookandfeel/contents/lockscreen/LockScreen.qml
index 8b01322..ab554bd 100644
--- a/lookandfeel/contents/lockscreen/LockScreen.qml
+++ b/lookandfeel/contents/lockscreen/LockScreen.qml
@@ -156,6 +156,14 @@ Image {
                             enabled: !authenticator.graceLocked
                             onAccepted: unlockFunction()
                             focus: true
+                            //HACK: Similar hack is needed in sddm loginscreen
+                            //TODO: investigate
+                            Timer {
+                                interval: 200
+                                running: true
+                                repeat: false
+                                onTriggered: passwordInput.forceActiveFocus()
+                            }
                             visible: block.mainItem.model.get(block.mainItem.selectedIndex)["showPassword"]
                             onVisibleChanged: {
                                 if (visible) {
